src/config/schema-transformer/to_bgp.py

```diff
@@ -640,7 +640,15 @@ def locate_routing_instance(self, rinst_name, service_chain=None):
 
         if old_rtgt:
             rt_key = "target:%s:%d" % (self.get_autonomous_system(), old_rtgt)
-            _vnc_lib.route_target_delete(fq_name=[rt_key])
+            try:
+                _vnc_lib.route_target_delete(fq_name=[rt_key])
+            except NoIdError:
+                _sandesh._logger.error(
+                    "NoIdError while deleting route target: ")
+            except RefsExistError:
+                _sandesh._logger.error(
+                    "Back referenes exists while deleting route target: ")
```

Comment:
Can you also print rinst_fq_name_str to identify which customer's rt might be in problem ?